Git Basic commands.txt

```diff
@@ -0,0 +1,128 @@
+
+                           𝐁𝐀𝐒𝐈𝐂 𝐆𝐈𝐓 𝐂𝐎𝐌𝐌𝐀𝐍𝐃𝐒
+                          _______________________
+
+1. git config --global user.name “[firstname lastname]”
+set a name that is identifiable for credit when review version history
+
+2. git config --global user.email “[valid-email]”
+set an email address that will be associated with each history marker
+
+3. git config --global color.ui auto
+set automatic command line coloring for Git for easy reviewing
+
+4. git init
+initialize an existing directory as a Git repository
+
+5. git clone [url]
+retrieve an entire repository from a hosted location via URL
+
+6. git status
+show modified files in working directory, staged for your next commit
+
+7. git add [file]
+add a file as it looks now to your next commit (stage)
+
+8. git reset [file]
+unstage a file while retaining the changes in working directory
+
+9. git diff
+diff of what is changed but not staged
+
+10. git diff --staged
+diff of what is staged but not yet commited
+
+11. git commit -m “[descriptive message]”
+commit your staged content as a new commit snapshot
+
+12. git init
+initialize an existing directory as a Git repository
+
+13. git clone [url]
```

Comment:
This one is also repeated, already on point 5

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Reusable